cmd2/argparse_custom.py

```diff
@@ -528,6 +532,39 @@ def _match_argument_wrapper(self, action, arg_strings_pattern) -> int:
 # noinspection PyProtectedMember
 argparse.ArgumentParser._match_argument = _match_argument_wrapper
 
+
+############################################################################################################
+# Patch argparse._SubParsersAction to add remove_parser function
+############################################################################################################
+
+def _SubParsersAction_remove_parser(self, name: str):
+    """
+    Removes a sub-parser from a sub-parsers group
+
+    This is a custom method being added to the argparse._SubParsersAction
+    class so cmd2 can remove subcommands from a parser.
+
+    :param self: instance of the _SubParsersAction being edited
+    :param name: name of the sub-parser to remove
+    """
+    for choice_action in self._choices_actions:
+        if choice_action.dest == name:
+            self._choices_actions.remove(choice_action)
```

Comment:
Recommend adding unit test where the code inside this `for` and inner `if` is executed

Reply:
As far as I can tell this is a bug in how coverage is being reported. I debugged and stepped through this and verified it is in fact getting hit.